### ISSUE TITLE
Observable.zip and Observable.prototype.zip have different interfaces

### ISSUE THREAD

**Issue by menelaos** [Reactive-Extensions/RxJS#867]
Follow-up to https://github.com/Reactive-Extensions/RxJS/issues/853

The following example still does not work despite being explicitly mentioned in
https://github.com/Reactive-Extensions/RxJS/releases/tag/v3.1.0 :

```javascript
var source = Rx.Observable.zip([
  Rx.Observable.of(1,2,3),
  Rx.Observable.of(4,5,6),
  Rx.Observable.of(7,8,9)
]);
//Uncaught TypeError: Cannot read property 'apply' of undefined
```

It only works correctly as an instance method:

```javascript
var source = Rx.Observable.of(1,2,3).zip([
  Rx.Observable.of(4,5,6),
  Rx.Observable.of(7,8,9)
]);
```

**Comment by mattpodwysocki** [comment]
@menelaos closed by 3.1.1

**Comment by menelaos** [comment]
Thanks!
